Documentation/network/servicemesh/ingress.rst

```diff
@@ -82,23 +86,20 @@ Supported Ingress Annotations
      - ``LoadBalancer``
    * - ``ingress.cilium.io/insecure-node-port``
      - | The NodePort to use for the HTTP Ingress.
-       | Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``. If unspecified, a random
+       | Applicable only if ``ingress.cilium.io/service-type``
+       | is ``NodePort``. If unspecified, a random
        | NodePort will be allocated by kubernetes.
      - unspecified
    * - ``ingress.cilium.io/secure-node-port``
```

Comment:
@mhofstetter Do we still need the differentiation between `ingress.cilium.io/secure-node-port` and `ingress.cilium.io/insecure-node-port` after merging https://github.com/cilium/cilium/pull/31646?